### PR TITLE
Pull Request Template added to the docs

### PR DESCRIPTION
**Pull request by roy-pon**
Section added on how a Pull Request Template can streamline the pull request process. And a optional Pull Request Template which can be used for this process.

### CODE REVIEW

**Review by pon-james — APPROVED** [review]
LGTM

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+## Related Tickets & Documents
+*Provide links to any related tickets or documents.*
+
+## QA Instructions, Screenshots, Recordings
+*Include the steps that need to be followed to test your changes. If applicable, include screenshots or recordings to help explain your PR.*
+
+## Checklist before requesting a review
+*Please leave a comment to the reviewer if something is not applicable.*
+I have:
+- [ ] Reviewed my own code
```
Comment (zeger-pon) [inline pull_request_template.md:32]: Only relevant files were touched

Comment (zeger-pon) [inline pull_request_template.md:32]: Only one feature/fix was added per PR -> if relevant?

Comment (zeger-pon) [inline pull_request_template.md:32]: Used the correct branch (or something like this)?

**Review by zeger-pon — CHANGES_REQUESTED** [review]

File: pull_request_template.md (hunk 1 above)
Comment (roy-pon) [reply pull_request_template.md:32]: @zeger-pon comments have been resolved, all good points of things we have seen happening in the past. So added them all.

**Review by zeger-pon — APPROVED** [review]

**Pull request merged**